src/inet/tests/TestInetCommonPosix.cpp

```diff
@@ -450,7 +450,11 @@ void ServiceEvents(struct ::timeval & aSleepTime)
         if (NetworkIsReady())
 #endif
         {
+#ifdef __MBED__
```

Comment:
Could we just not write pid data for any of the variants?
I am unsure if it is used and it seems like debug code only. I would prefer less `#ifdef`

Reply:
Yes, we were surprised of having these prints. It's not like `getpid` and `getppid` are very common on embedded targets.

Reply:
Yes, we were surprised of having these prints. It's not like `getpid` and `getppid` are very common on embedded targets.

Reply:
I'd be in favor of just not printing pid/ppid here.

Reply:
I removed the printing.